packages/web-frontend/src/containers/Map/Map.js

```diff
@@ -68,7 +68,8 @@ const StyledMap = styled(LeafletMap)`
  */
 class MapComponent extends Component {
   componentWillMount() {
-    window.addEventListener('resize', () => this.forceUpdate());
+    // ensure leaflet updates after 0.5 second resize animation has finished
+    window.addEventListener('resize', () => setTimeout(() => this.map?.invalidateSize(), 500));
```

Comment:
Changes in this file mainly represent the fact that we're wrapping the map in an outer div, which removes a couple of funky things that were going on:
- ZoomControl had to reposition itself based on the side panel width
- The ui-components leaflet map had to know what the side panel width was, and adjust coordinates based on that

It does add this one funky thing though as a tradeoff, which is that we need to invalidate the leaflet map after the resize animation of its wrapping container has finished